erica/api/v2/endpoints/est.py

```diff
@@ -40,8 +45,8 @@ async def get_send_est_job(request_id: UUID):
     :param request_id: the id of the job.
     """
     try:
-        erica_request = get_erica_request(request_id)
-        return create_response(erica_request)
+        tax_declaration_service: TaxDeclarationServiceInterface = injector.inject(TaxDeclarationServiceInterface)
```

Comment:
Why do we moved the injector into the class? The injector the send_est should not use the injector directly instead we should use a method or a parameter. Wdyt?

Reply:
We had the injector before in the utils class and we were calling from the API endpoints that utils class. But now we have a service interface for every use case and therefore it is (for now) being injected in every endpoint.
But we could maybe moved that service definition to the injector factory?